### ISSUE TITLE
/waiver endpoint 

### ISSUE THREAD

**Issue by avsomers25** [HackRU/frontendv2#44]
The exists a card in the page.tsx file in the dashboard folder, in the top right of the page, where there is a file upload input  for the waiver 

in order to get to the /dashboard page, it will redirect you to /login the email is test@test.org and the password is test, it won't redirect you but it will then allow you to go to /dashboard

this input, along with the register button should be wrapped in a form, similar to how the card below is wrapped in a form
this form should have some form of validation, which prevents the register button from being submitted unless a waiver is uploaded

once there is a waiver, you should be able to click the register button and it will trigger some function in data.ts, which is in the lib folder this function should then call two function, GetWaiverInfo and UploadWaiver, and handle the response from them 

these function are already implement in js in the old frontend repository, which you can find under the HACKRU github. And should be created in the frontendv2 repo in the action.ts file, similar to how the other endpoint functions were implemented in action.ts 

In summary, what you are creating is a form that takes a waiver file as a required input, and then on submitting of the form uploads the waiver  If you have any question just message Andrew on discord and I'll be happy to help you out!
